### PR TITLE
fix: support int64 indices for embedding

### PR DESCRIPTION
**Pull request by fs-eire**
# Description

Allow embedding conversion to accept TensorRT int32 and int64 indices.

There used to be a restriction that TensorRT v8.x only works with int32 indices for `embedding` op, but since TensorRT v9, int64 is supported. We no longer need to reject int64 indices now.

Fixes # (issue)

## Type of change

- Bug fix (non-breaking change which fixes an issue)

# Checklist:

- [x] My code follows the style guidelines of this project (You can use the linters)
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas and hacks
- [ ] I have made corresponding changes to the documentation
- [x] I have added tests to verify my fix or my feature
- [x] New and existing unit tests pass locally with my changes
- [ ] I have added the relevant labels to my PR in so that relevant reviewers are notified

### CODE REVIEW

File: tests/py/dynamo/conversion/test_embedding_aten.py [hunk 1]
```diff
@@ -29,6 +29,18 @@ class TestEmbeddingConverter(DispatchTestCase):
...
+                indices_tensor=torch.tensor([3, 1, 2], dtype=torch.int64),
+                weights_tensor=torch.randn((5, 10), dtype=torch.float32),
+                sparse=False,
+            ),
+            param(
+                test_name="2d_indices_int64",
+                indices_tensor=torch.tensor([[3, 1, 2], [4, 1, 3]], dtype=torch.int64),
+                weights_tensor=torch.randn((5, 10), dtype=torch.float32),
+                sparse=True,
+            ),
```
Comment (micwill755) [inline tests/py/dynamo/conversion/test_embedding_aten.py:43]: We should also add a 3D int64 indices test case, to match the existing int32 3d_indices coverage.

**Review by zewenli98 — APPROVED** [review]
LGTM. waiting for CI

File: tests/py/dynamo/conversion/test_embedding_aten.py (hunk 1 above)
Comment (fs-eire) [reply tests/py/dynamo/conversion/test_embedding_aten.py:43]: test case added.
